### PR TITLE
Adding util for triggering build with no backward compatibility check

### PR DESCRIPTION
**Pull request by ronykoz**
## Status
Ready

Adding util for triggering build with no backward compatibility check

### CODE REVIEW

File: Utils/trigger_content_build_with_no_backward_check.sh [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+
+_branch=$1
+_circle_token=$2
+
+trigger_build_url=https://circleci.com/api/v1/project/demisto/content/tree/${_branch}?circle-token=${_circle_token}
+
+post_data=$(cat <<EOF
+{
+  "build_parameters": {
+    "BACKWARD_COMPATIBILITY": "true"
```
Comment (yaakovi) [inline Utils/trigger_content_build_with_no_backward_check.sh]: shouldn't this be false?
we want to trigger the build without the BC checks

**Comment by ronykoz** [comment]
@yuvalbenshalom @yaakovi review me

**Comment by yaakovi** [comment]
on second look, i think the parameter should be passed to the actual validation of the BC and skip/ignore the result.
in the current way, this will ignore other problems that not related to BC (secrets, structure, image size, etc.)

File: Tests/scripts/validate_files.py [hunk 2]
```diff
@@ -140,7 +140,7 @@ def validate_modified_files(self, modified_files):
                 old_file_path, file_path = file_path
 
             print("Validating {}".format(file_path))
-            structure_validator = StructureValidator(file_path, is_added_file=False,
+            structure_validator = StructureValidator(file_path, is_added_file=not(False and is_backward_check),
```
Comment (yaakovi) [inline Tests/scripts/validate_files.py]:
```suggestion
            structure_validator = StructureValidator(file_path, is_added_file=not(False or is_backward_check),
```

**Comment by yaakovi** [comment]
with BC check:
image


without BC check:
image

**Pull request merged**